pycqed/measurement/calibration_points.py

```diff
@@ -19,10 +19,12 @@ def __init__(self, qb_names, states, **kwargs):
         self.states = states
         default_map = dict(g=['I '], e=["X180 "], f=['X180 ', "X180_ef "])
         self.pulse_label_map = kwargs.get("pulse_label_map", default_map)
+        self.prep_params = None
```

Comment:
Is there a specific reason why you have the prep params as a state for the cal_point object? In principle, a cal point could remain independent of the preparation and could be used with various preparations, right? We could in that case have the prep_params as an argument of the get_indices() function, which, as you pointed out, needs to know which preparation was made. What do you think? 
One reason for which I imagine you could have done it in that way is to save the prep_params as part of the cal point object (?). Ideally, but I guess we should be able to retrieve them from the hdf5 file in any case.